controller/workitem.go

```diff
@@ -601,6 +601,34 @@ func workItemIncludeHasChildren(appl application.Application, ctx context.Contex
 	}
 }
 
+// includeParentWorkItem adds the parent of given WI to relationships & included object
+func includeParentWorkItem(ctx context.Context, appl application.Application) WorkItemConvertFunc {
+	return func(request *http.Request, wi *workitem.WorkItem, wi2 *app.WorkItem) {
+		var parentID *uuid.UUID
+		var err error
+		repo := appl.WorkItemLinks()
+		if repo != nil {
+			parentID, err = repo.GetParentID(ctx, wi.ID)
+			if err != nil {
+				log.Error(ctx, map[string]interface{}{
```

Comment:
This could be a `Info` or `Debug` info, not necessarily an `Error`

Reply:
Yes! using `Info` in https://github.com/fabric8-services/fabric8-wit/pull/1592/commits/f267820f7c360f8da54ca40e6e34d6399f38f77e